service/github/github.go

```diff
@@ -302,10 +303,30 @@ func (g *PullRequest) setPostedComment(ctx context.Context) error {
 func (g *PullRequest) Diff(ctx context.Context) ([]byte, error) {
 	opt := github.RawOptions{Type: github.Diff}
 	d, _, err := g.cli.PullRequests.GetRaw(ctx, g.owner, g.repo, g.pr, opt)
+	if err == nil {
+		return []byte(d), nil
+	}
+
+	pr, _, err := g.cli.PullRequests.Get(ctx, g.owner, g.repo, g.pr)
 	if err != nil {
 		return nil, err
```

Comment:
https://github.com/reviewdog/reviewdog/blob/7644825553f0a1b7681448dce36f8dc7ac16d686/service/github/github.go#L305-L308
Is this a review of the above code?

Reply:
I think that this line of code is a part of the git command flow.

Reply:
Oh my bad, I misread it. 
Can you address the second point?

Reply:
>I think that this line is a part of the git command flow.

https://github.com/reviewdog/reviewdog/blob/86c205edb68bba175f6dca979d71fd2d772c4126/service/github/github.go#L304-L308
This is the code of GitHub API flow.

https://github.com/reviewdog/reviewdog/blob/86c205edb68bba175f6dca979d71fd2d772c4126/service/github/github.go#L310-L329
This is the code of git command flow.

I wanted to say that you reviewed about the GitHub API flow while you pointed to the code in the git command flow.

Reply:
https://github.com/reviewdog/reviewdog/blob/b47e57c7e28eab11441fa3f506827f8c7f77d15e/service/github/github.go#L305

I fixed the above code to fallback on 406 error and return err otherwise: https://github.com/reviewdog/reviewdog/pull/1697/commits/b47e57c7e28eab11441fa3f506827f8c7f77d15e